nfpm.yaml

```
@@ -0,0 +1,61 @@
+# nfpm configuration file
+#
+# check https://nfpm.goreleaser.com/configuration for detailed usage
+#
+name: "modbus-plugin"
+arch: "all"
+platform: "linux"
+version: "0.0.1"
+section: "default"
+priority: "extra"
+maintainer: techforum.softwareag.com
+description: "Modbus plugin for the thin-edge.io"
+homepage: "https://github.com/thin-edge/modbus-plugin"
+license: "Apache-2.0"
+replaces:
+#- modbus-plugin
+provides:
+#- modbus-plugin
+depends:
+  - python (>= 3.8.0) | python3 (>= 3.8.0)
```

You should also add a dependency to `python3-pip` as pip is not installed by default on debian systems.

```suggestion
  - python (>= 3.8.0) | python3 (>= 3.8.0)
  - python3-pip
```